core/src/main/java/org/mskcc/cbio/oncokb/model/Evidence.java

```diff
@@ -31,7 +31,6 @@
     private String knownEffect;
     private Date lastEdit;
     private LevelOfEvidence levelOfEvidence;
-    @JsonIgnore
```

Comment:
Adding JsonIgnore annotation for propagation will cause not being able to pass propagation from the front end evidence model.